src/components/flag/flag.scss

```diff
@@ -0,0 +1,20 @@
+.flag-img {
+  object-fit: cover;
+  img {
+    height: 100%;
+  }
+}
+
+@media (max-width: 475px) {
+  .flag-img {
+    width: 180px;
+    height: 108px;
+  }
+}
+
+@media (max-width: 375px) {
+  .flag-img {
+    width: 150px;

```

Comment:
Setting width/height with CSS for the flag image conflicts with approach to set width and height with props value. Please use one consistent solution for desktop and mobile.
I would recommend you introduce property `size` witch could be `['LARGE', 'MEDIUM', 'SMALL']` and apply width based on the value of that property